app/views/app_inspector/index.html.erb

```diff
@@ -0,0 +1,69 @@
+<h1>App Inspector</h1>
+<select name="app_inspector_selector" id="app_inspector_selector">
+  <% @app_names.each do |app_name| %>
+    <option value="<%= app_name %>"><%= app_name %></option>
+  <%end %>
+</select>
+<div id="app_inspector_semi_outer">
+  <p id="y_label">Frequency</p>
+  <div id="app_inspector_inner">
+    <div id="app_inspector_bar_1" class="app-inspector-bar"></div>
+  </div>
+</div>
+<p id="x_label">Cores per job</p>
+<input id="app_inspector_bins_slider" type="range" min="1" max="20" value="5">
+
+<script>
+  appInspectorData = <%= @app_inspector_data.to_json.html_safe %>
+  innerBox = document.getElementById("app_inspector_inner");
```

Comment:
Don't use javascript (or lots of). I want to really buy into Rails libraries. One is turbo-streams to be able to render and change panels server side.  We should be writing some amount of `turbo_stream.erb` extension templates.

So that's the server side. The server knows how to respond to turbo-stream requests.

Client side we can use `stimulus` to tack on simple controllers to make these turbo stream requests.  So we have erb templates for streams and the controllers to respond to it.  A very simple `stimulus` can bind to the input `app_inspector_bins_slider` and then request an outright replace with `{ bins: 10, app: 'jupyter}` (how many bins and what app to render)